queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -95,6 +97,15 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
             'dissolutionDate': LegislationDatetime.format_as_legislation_date(business.dissolution_date)
         }
 
+    # update batch processing entry, if any is present
+    if flag_on:
+        batch_processings = BatchProcessing.find_by(filing_id=dissolution_filing.get("id"))
```

Comment:
I don't think the filing json will have the filing id?

maybe try using `filing_rec.id`?

Reply:
Yep, try to use `filing_rec.id`. And I have one more question, do we know if filing_id must be saved into batch_processing before filer processes it? I notice [it](https://github.com/bcgov/lear/blob/911b7ef4efa5168e97fa7108df2d97f7689e665d/jobs/involuntary-dissolutions/involuntary_dissolutions.py#L294) is saved after the message is pushed into queue (though I know filer will process it async)

Reply:
I think it's probably a good call to save the filing_id to the batch_processing before putting it on the queue. In theory the filer won't process it right away, which will give the job a chance to save the batch_processing entry (this was my experience testing locally). Still better to be safe imo so I've updated the job.